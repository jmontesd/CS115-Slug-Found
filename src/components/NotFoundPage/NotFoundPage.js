import React from 'react';
import './NotFoundPage.scss';
import { Link } from 'react-router-dom';

const NotFoundPage = () => {
  return (
    <div className="container">
      <Link to="/">404 - Go home</Link>
    </div>
  );
};

export default NotFoundPage;
