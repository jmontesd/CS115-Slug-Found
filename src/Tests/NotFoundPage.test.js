import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { NotFoundPage } from '../components/NotFoundPage/NotFoundPage';

// needed for shallow rendering
configure({ adapter: new Adapter() });

describe('<NotFoundPage />', () => {
  it('should render NotFoundPage properly', () => {
    // test creates snapshot of component and checks if matches
    // previously create snapshot
    const wrapper = shallow(<NotFoundPage isLoggedIn />);
    expect(wrapper).toMatchSnapshot();
  });
});
