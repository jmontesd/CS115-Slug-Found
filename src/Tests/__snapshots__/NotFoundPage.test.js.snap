// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should render NotFoundPage properly 1`] = `ShallowWrapper {}`;
